Sources/Noora/NooraMock.swift

```diff
@@ -0,0 +1,53 @@
+#if DEBUG
+    import Rainbow
+
+    public class MockNoora: Noora, CustomStringConvertible {
+        public var description: String {
+            standardPipelineEventsRecorder.events.map { event in
+                event.content.split(separator: "\n")
+                    .map {
+                        "\(event.type): \($0)"
+                    }.joined(separator: "\n")
+            }.joined(separator: "\n")
+        }
+
+        var standardPipelineEventsRecorder = StandardPipelineEventsRecorder()
+
+        public init() {
+            super.init(standardPipelines: StandardPipelines(
+                output: MockStandardPipeline(type: .output, eventsRecorder: standardPipelineEventsRecorder),
+                error: MockStandardPipeline(type: .error, eventsRecorder: standardPipelineEventsRecorder)
+            ))
+        }
+    }
+
+    public class StandardPipelineEventsRecorder {
```

Comment:
What do you think if we place all these utilities inside `MockNoora` and remove `Mock` from the names? The symbols would become:

```swift
MockNoora.StandardPipelineEventsRecorder
MockNoora.StandardOutputEvent
MockNoora.StandardPipelineType
MockNoora.StandardPipeline
```